src/js/apps/patients/schedule/schedule_app.js

```diff
@@ -6,7 +6,7 @@ import App from 'js/base/app';
 
 import intl, { renderTemplate } from 'js/i18n';
 
-import StateModel from './schedule_state';
+import StateModel, { STATE_VERSION } from './schedule_state';
```

Comment:
I moved state version to an export because it took me a while to realize I had to update it in two places